### PR TITLE
Release

### PR DESCRIPTION
**Pull request by tab-cmd**
# Overview

Added a changelog for the upcoming release!

## Ticket

https://www.pivotaltracker.com/story/show/181038279
https://www.pivotaltracker.com/story/show/181038264

## Contributions

- `README.md`: with up to date information
- `setup.py`: removed unsupported python versions and adjust the version tag 
- `CHANGELOG.md`: with highlights and detailed changes (somewhat!)

## Test

- N/A

## Documentation

- Are documentation updates required? In-line, README, or [documentation](https://github.com/BciPy/bcipy.github.io)? Verify the updates you did here.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -17,3 +17,7 @@ Link a pivotal ticket here
 ## Documentation
 
 - Are documentation updates required? In-line, README, or [documentation](https://github.com/BciPy/bcipy.github.io)? Verify the updates you did here.
+
+## Changelog
+
+- Is the CHANGELOG.md updated with your detailed changes?
```
Comment (lawhead) [inline .github/pull_request_template.md:23]: This is a good callout.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,39 +2,93 @@
...
+- `signal/model/base_model.py`: add reshaper to base model class, requiring it to be defined in all models and return a `Reshaper`. Fix return types. #147
+- `signal/model/offline_analysis.py`: updated to use new reshapers and transforms. #147 updated to report execution time and logging levels #163
+- `bcipy/language_model/` --> `bcipy/language/` refactor for clarity and add base class `LanguageModel` #164
+- `bcipy/tasks` --> `bcipy/task`: refactor for clarity, add README, organize tasks under paradigm #162 organize tasks operation objects into `control` module #162 #178
+- `task/paradigm/rsvp/copy_phrase.py`: refactored overall #146 to use new Session classes #127 and updated to use new reshapers and transforms #147 implements current state of inquiry preview #129 #177 to account for max selection parameter #175 fix targetness #179
+- `bcipy/task/data.py`: to track number of decisions made #175
+- `task/control/handler.py`: added the ability to set constants in defined stimuli agent #178
+- `task/control/query.py`: emove redundant best_selection in favor of one with constants. Implemented constants in return_stimuli methods. #178
```
Comment (lawhead) [inline CHANGELOG.md]: Looks like there is a missing `r` at the start of the sentence.

File: README.md [hunk 3]
```diff
@@ -8,37 +8,17 @@
...
-- Do not push IDE or other local configuration files.
-- All new modules or major functionality should be documented outside of the code with a README.md.
-	See README.md in repo or go to this site for inspiration: https://github.com/matiassingers/awesome-readme. Always use a Markdown interpreter before pushing. There are many free online or your IDE may come with one.
-
-See this resource for examples: http://docs.python-guide.org/en/latest/writing/style/
-
 ## Dependencies
 ---------------
-This project requires Python > 3.6.5 and other packages defined in the requirements.txt.
+This project requires Python 3.7 or 3.8. All other dependancies defined in the requirements.txt.
```
Comment (lawhead) [inline README.md]: Typo: should be 'dependencies'.

**Review by lawhead — APPROVED** [review]
This was a lot of work! Thanks.

File: README.md (hunk 3 above)
Comment (tab-cmd) [reply README.md]: I will never learn to spell haha thanks for the catch

**Pull request merged**